codejam-canvas/32x32.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CodeJam - Canvas</title>
+    <link rel="stylesheet" href="./main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <nav>
+                <img src="./assets/nav.svg" alt="">
+            </nav>
+            <h1>CodeJam - Pallete</h1>
+        </div>
+        
+        <img src="./assets/menu icon.svg" alt="" class="menu-icon">
+    </header>
+    <main>
+        <section>
+            <ul>
+                <li><img src="./assets/paint bucket.svg" alt="">
+                    Paint bucket
+                </li>
+                <li>
+                    <img src="./assets/eye-dropper.svg" alt="">
+                    Choose color
+                </li>
+                <hr>
+                <li>
+                    <img src="./assets/move.svg" alt="">
+                    Move
+                </li>
+                <li>
+                    <img src="./assets/transform.svg" alt="">
+                    Transform
+                </li>
+            </ul>
+            <ul>
+                    <li>
+                        <img src="./assets/current color.svg" alt="">
```

Comment:
in this task this img is OK, but in other tasks you need to add functionality with changing color, consequently img will not proper solution

Reply:
make circles with CSS